docs/Manifest.toml

```diff
@@ -1,23 +1,58 @@
 # This file is machine-generated - editing it directly is not advised
```

Comment:
Yes, of course, it's used for instantiating the environment when building the documentation. See eg https://discourse.julialang.org/t/psa-use-a-project-for-building-your-docs/14974/7.

Reply:
OK. I tend to prefer using the latest versions, so that we see what users would see. Otherwise we're likely to forget updating these (we usually forget to upgrade Documenter itself...). Anyway, that's another issue.